rfc/rfc-xxxx-template.md

```diff
@@ -0,0 +1,34 @@
+# RFC-#: Title
+
+**Authors:**
+
+**Status:** **Active** | Accepted | Rejected | Superseded
+
+**Pull Request URL:** link
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+
+## Problem
+*What is this proposal solving? Why does the problem matter? How does the problem affect the user?*
+
+### Anti-Goals
+*What is outside the scope of what the proposal is trying to solve?*
+
+## Solution
+*Describe your solution and how it’s going to solve the problem. This is likely the largest section of your proposal and might even include some high-level diagrams if you are proposing code changes. While all important aspects need to be covered, also keep in mind that shorter documents are more likely to be read.*
+
+### User Impact
```

Comment:
@trisberg had a good question as to what "User" means

See https://github.com/projectriff/riff/pull/1360#discussion_r346862796

Reply:
I think we should consider the impact of the rfc on all the users, both operators and function developers.